spec/features/tagging_content_spec.rb

```diff
@@ -1,6 +1,15 @@
 require "rails_helper"
 
-RSpec.describe "Tagging content" do
+RSpec.describe "Tagging content", type: :feature do
+  include PublishingApiHelper
+
+  before do
```

Comment:
There's a bit of a trade off here, but I generally prefer to avoid "mystery guests":

https://robots.thoughtbot.com/mystery-guest

In this test, we start seeing a "magic" thing named "Business tax / Pension scheme administration". The reader would have to have the knowledge that this is a side effect from `publishing_api_has_topics` to understand that.

There are two ways of preventing it:

1. Move the entire `PublishingApiHelper` into this class
2. Explicitly create the data we need for this test via the test helper (`publishing_api_has_topics(title: "Business tax / Pension scheme administration")` and have the test set the whole thing up properly